### DISCUSSION TITLE
Writing up assignment - Team Work parts

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#30] [General]
Hi all,

Tonight I have a Python assignment to finish but I will finish writing everything up by tomorrow evening.

Thanks for your collaboration in this "adventure". 

Regards,

Mark

**Comment by StephenHillenbrand** [comment]
How you going Mark?. If you need a hand please email. I'll be up late watching the race at Monza.

**Comment by MarkW06071969** [comment]
Nearly finished, just typing up Section G now. Will then go over everything again to check everything is there.
Enjoy the race! 👍

Sent from Yahoo Mail on Android 
 
  On Sun, 11 Sept 2022 at 7:55 pm, ***@***.***> wrote:   


How you going Mark?. If you need a hand please email. I'll be up late watching the race at Monza.
